homeassistant/components/tuya/humidifier.py

```diff
@@ -40,7 +40,7 @@ class TuyaHumidifierEntityDescription(HumidifierEntityDescription):
         humidity=DPCode.DEHUMIDITY_SET_VALUE,
         device_class=HumidifierDeviceClass.DEHUMIDIFIER,
     ),
-    # Humidifier
+    # Humidifier & Diffuser
```

Comment:
This is incorrect, Diffusers are category `xxj`, not `jsq`, please revert this comment.